File: wwld_backend/demo/src/main/java/com/gateway/entity/Concept.java
package com.gateway.entity;


import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter

@Entity
@Table(name="concept")
public class Concept {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name="id")
    private long id;

    @Column(name="title")
    private String title;

    @Column(name="slug")
    private String slug;

    @Column(name="contentMd",columnDefinition = "TEXT")
    private String contentMd;

    @Column(name="concept_image")
    private String conceptImage;

    @Column(name="description",columnDefinition = "TEXT")
    private String description;
}

File: wwld-next-app/src/lib/types/concept.ts
// Concept interface payload
export interface ConceptPayload{
    id?: number;
    title: string;
    slug: string;
    contentMd?: string;
    conceptImage?: string;
    description? : string;
}


export interface Concept{
  id?: number;
  title: string;
  slug: string;
  contentMd?: string;
  conceptImage?: string;
  description?: string;
};